testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/AbstractAdminTest.java

```diff
@@ -71,6 +75,16 @@ public void addTestRealms(List<RealmRepresentation> testRealms) {
         config.put("port", "3025");
         adminRealmRep.setSmtpServer(config);
 
+        // add dummy federation provider
```

Comment:
Don't add a dummy federation provider here as it affects a large amount of other tests. Use the admin client in the one test you need it in to add it